### PR TITLE
QE: Replace CentOS8 with Rocky 8 in BV

### PR DESCRIPTION
**Pull request by nodeg**
## What does this PR change?

This will replace CentOS 8 with Rocky 8 in our build validation.

## GUI diff

No difference.

- [x] **DONE**

## Documentation

- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
- Cucumber tests were adjusted

- [x] **DONE**

## Links

- Fixes https://github.com/SUSE/spacewalk/issues/18030
- Manager 4.3: https://github.com/SUSE/spacewalk/pull/18611
- Manager 4.2: https://github.com/SUSE/spacewalk/pull/18612

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by nodeg** [comment]
image

**Review by Bischoff — APPROVED** [review]

File: testsuite/features/build_validation/add_custom_repositories/add_rocky8_repositories.feature [hunk 1]
```diff
@@ -1,65 +1,65 @@
...
+    Then I should see a "Channel Custom Channel for Rocky 8 DVD created" text
 
-  Scenario: Add the CentOS 8 Appstream DVD repository
+  Scenario: Add the Rocky 8 Appstream DVD repository
     When I follow the left menu "Software > Manage > Repositories"
     And I follow "Create Repository"
-    And I enter "centos-8-iso-appstream" as "label"
-    And I enter "http://127.0.0.1/centos-8-iso/AppStream" as "url"
+    And I enter "rocky-8-iso-appstream" as "label"
+    And I enter "http://127.0.0.1/rocky-8-iso/AppStream" as "url"
```
Comment (srbarrios) [inline testsuite/features/build_validation/add_custom_repositories/add_rocky8_repositories.feature:30]: Did you confirmed that it follows the same paths "AppStream" and "BaseOS" ?

File: testsuite/features/build_validation/reposync/srv_sync_all_products.feature [hunk 2]
```diff
@@ -133,18 +133,18 @@ Feature: Synchronize products in the products page of the Setup Wizard
...
-    Then I should see the "SUSE Linux Enterprise Server with Expanded Support 8" selected
+    And I enter "Rocky Linux 8" as the filtered product description
+    And I select "Rocky Linux 8 x86_64" as a product
+    Then I should see the "Rocky Linux 8 x86_64" selected
+    When I open the sub-list of the product "Rocky Linux 8 x86_64"
+    And I select "SUSE Manager Client Tools for RHEL and ES 8 x86_64" as a product
+    Then I should see the "SUSE Manager Client Tools for RHEL and ES 8 x86_64" selected
     When I click the Add Product button
-    And I wait until I see "RHEL or SLES ES or CentOS 8 Base" product has been added
+    And I wait until I see "Rocky Linux 8 x86_64" product has been added
```
Comment (srbarrios) [inline testsuite/features/build_validation/reposync/srv_sync_all_products.feature:147]: No no.
We don't want that. Not for this PR at least.
We want to keep testing RES (RH Expanded Support) product, the one created and handled by SUSE, using the DVD image of a Rocky8.
We don't want to test Rocky8 Pure as the goal of this PR. Even if we might consider that other additional client in the future.

Comment (srbarrios) [reply testsuite/features/build_validation/reposync/srv_sync_all_products.feature:147]: **Clarifying**:
All RHEL-like products can be registered using their Pure SCC products or using RES (A special subscription created by SUSE to handle that kind of Distros).
We should cover both options, but for now, we want to assure to cover RES.

File: testsuite/features/build_validation/add_custom_repositories/add_rocky8_repositories.feature (hunk 1 above)
Comment (nodeg) [reply testsuite/features/build_validation/add_custom_repositories/add_rocky8_repositories.feature:30]: Yes.
```bash
❯ sudo mount Rocky-8-x86_64-dvd1.iso /mnt
[sudo] password for root: 
mount: /mnt: WARNING: source write-protected, mounted read-only.

❯  ll /mnt/
total 14
dr-xr-xr-x 4 root root 2048 2022-05-15 23:06 AppStream
dr-xr-xr-x 4 root root 2048 2022-05-15 23:06 BaseOS
dr-xr-xr-x 3 root root 2048 2022-05-15 23:06 EFI
dr-xr-xr-x 3 root root 2048 2022-05-15 23:06 images
dr-xr-xr-x 2 root root 2048 2022-05-15 23:06 isolinux
-r--r--r-- 1 root root 2204 2022-03-30 02:02 LICENSE
-r--r--r-- 1 root root   86 2022-05-15 23:05 media.repo
-r--r--r-- 1 root root  883 2022-05-15 23:06 TRANS.TBL

❯  ll /mnt/AppStream/
total 8
dr-xr-xr-x 29 root root 4096 2022-05-15 23:06 Packages
dr-xr-xr-x  2 root root 4096 2022-05-15 23:06 repodata
dgedon@b660:~/isos> ll /mnt/BaseOS/
total 8
dr-xr-xr-x 28 root root 4096 2022-05-15 23:06 Packages
dr-xr-xr-x  2 root root 4096 2022-05-15 23:06 repodata
```

File: testsuite/features/build_validation/reposync/srv_sync_all_products.feature (hunk 2 above)
Comment (nodeg) [reply testsuite/features/build_validation/reposync/srv_sync_all_products.feature]: Thanks for the clarificaiton. I reverted those changes.

**Review by srbarrios — APPROVED** [review]

**Pull request merged**